core/calico/api-server-release.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: release-calico-api-server
+  namespace: flux-config
+spec:
+  chart:
+    spec:
+      chart: adp-calico-helm-chart
+      version: "0.0.1"
```

Comment:
sem-ver is 0.1.0

Reply:
@DaRoszko PR Created to update helm version to 0.1.0.  https://github.com/DEFRA/adp-core-helm-charts/pull/2.  I'll update this once that is approved.

Reply:
@DaRoszko - Updated to 0.1.0